keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -207,54 +199,55 @@ Yes, the feature gate can be disabled, but Service resource that have set the ne
 
 * **What happens if we reenable the feature if it was previously rolled back?**
 
-New Services should be able to set the `trafficPolicy` field. Existing Services that have the field set already should not be impacted.
+New Services should be able to set the `internalTrafficPolicy` field. Existing Services that have the field set will begin to apply the policy again.
 
 * **Are there any tests for feature enablement/disablement?**
 
-There will be unit tests to verify that apiserver will drop the field when the `ServiceTrafficPolicy` feature gate is disabled.
+There will be unit tests to verify that apiserver will drop the field when the `ServiceInternalTrafficPolicy` feature gate is disabled.
 
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
 
-TBD for beta.
+Rollout should have minimal impact because the default value of `internalTrafficPolicy` is `Cluster`, which is the default behavior today.
 
 * **What specific metrics should inform a rollback?**
 
-TBD for beta.
+N/A
```

Comment:
Probably the blackhole metrics should be used here.

Reply:
done